settings/DevHome.Settings.UnitTest/TestingScenarios/Settings.md

```diff
@@ -0,0 +1,12 @@
+# Settings Tests
+If your code affects the settings pages, please manually verify these scenarios.
+
+## Scenarios
+Please make sure to verify all these scenarios.
+
+1. User can submit feedback via the feedback button
+1. User can set the app theme

```

Comment:
I assume this was supposed to be an ordered list (just a random find while browsing sorry for chiming in after merge 😂)